EDI/Overview/03_TipsTricks.md

```diff
@@ -0,0 +1,74 @@
+# Tips and tricks for preparation
+
+
+
+## Introduction
+
+UN/EDIFACT (in the following short EDIFACT) is a cross-industry international standard for the format of electronic data in business transactions. EDIFACT is one of several international EDI standards. A UN organization called CEFACT, which is affiliated to the UNECE, is responsible for the EDIFACT standard.
+[^1]    
+
+The EDIFACT standard defines a scheme according to which messages should be structured. It is used for data exchange and transfer between business partners, for example invoice or order data. The standard defines a list of messages such as ORDERS, INVRPT, INVOIC, DELFOR, etc. which are used for specific purposes. For example, an INVRPT represents an inventory report.   
```

Comment:
Vorschlag: It is used to exchange and transfer data between business partners, for example invoice or order data.
Oder umformulieren -> ... data, for example invoice or order data. (Beispiele beziehen sich auf data, aber neben business partners ist es verwirrend).

Reply:
Done